src/instabot.py

```diff
@@ -300,15 +301,15 @@ def catch_up_operations(self):
 
     def banned_sleep(self):
         # sleep for x minutes and try again
-        
+
         if self.banned['400'] < 3:
             sleep_time = 5*60
 
         else:
             sleep_time = 60*60
             self.banned['banned'] = True
             self.banned['400'] = 0
-        self.console.log('\tSleeping for %s minutes... '%(sleep_time/60))
+        self.console.log('\n\t--- Sleeping for %s minutes ---'%(sleep_time/60))
```

Comment:
Better logging indeed

Reply:
Not better, but clearer in console (for me at least)